sys/include/universal_address.h

```diff
@@ -28,30 +28,30 @@ extern "C" {
 #include <stdint.h>
 #include <stdlib.h>
 
-#define UNIVERSAL_ADDRESS_SIZE (16)         /**< size of the used addresses in bytes        */
+#define UNIVERSAL_ADDRESS_SIZE (16)         /**< size of the used addresses in bytes */
 
 /**
  * @brief The container descriptor used to identify a universal address entry
  */
 typedef struct universal_address_container_t {
     uint8_t use_count;                       /**< The number of entries link here */
-    uint8_t address_size;                    /**< Size in bytes of the used genereic address */
-    uint8_t address[UNIVERSAL_ADDRESS_SIZE]; /**< the genereic address data */
+    uint8_t address_size;                    /**< Size in bytes of the used generic address */
+    uint8_t address[UNIVERSAL_ADDRESS_SIZE]; /**< The generic address data */
 } universal_address_container_t;
 
 /**
- * @brief initializes the datastructure for the entries
+ * @brief Initializes the data structure for the entries
```

Comment:
While you're at it: didn't we conclude that the third person 's' should be omitted?